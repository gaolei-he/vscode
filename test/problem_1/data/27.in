100 5 6
kf heynhyjls
eft soioquip
ehovtn mu
qsecd jrgzuehfuj
pi sofgd
uolst eohyd
crnrlmad wq
mcpqrgf tcdkrsanr
zknvrmcx tpfs
cvxmyajmf dxsdwycw
zssvjtbkf ohk
ydwjgfpdrg ue
odxpvatab hgm
kl qhf
nwknzbyb lsofoqvma
szd mmtumzpktv
czdvpfcfb i
mjnfoge enjjkmo
dyox wlzxapog
fwpjpqenie gnjedre
imae oqhzcqbqx
dlsrnlh gftrcnq
gzk r
orfqpfvzn ymxl
durpdb c
aizdlaq msklkmwer
ypvtdveg hjxto
fwgokqg z
zkcjfpnfn hqtomegb
xoxuuxy ougpkjzxf
oddz aed
x q
bd sxjr
fdpth vtgqjfek
lextffyzw fwopza
ytppfqtmn tde
qgj xakodwgbxz
dwuneonh lnaqosf
qkzst ndzz
tbeckbc m
cmnrne ezp
uo fuqa
bkcqxkd xxokszh
dslqsjuw feufu
pasfjzb lixl
uayjyip ozduthqf
wpuif lfonrgpv
tjwgjmil jbroldxh
spfpz tlrquuvmbv
qyvgb fx
yvwf rhwezslli
xb s
hmnsptybct jbdvueb
ukbefigadr zcwivbh
ptrcnstl uxqswdem
mbmuccgjvt maksabrnzf
hvqmpo mxdiefabvf
marfaav ghq
ryhwv kjrprzpyg
vjinpehodz mgdg
jkndyaaufq j
gnsmes zhiepzitf
cke wtz
ccekxyuc glqmdmx
jfalxr rnqjggu
bllzxjvs drl
acuzmuq iudsxzd
naygip lsvfeocek
raplvroiuf momjkrshk
yswulbh dqppx
cj l
lomxcojoow wevt
yg vulvuw
ztygwg gyd
ipuffds ebcklwfg
mg zonbdaa
mt yrqooimnb
eieesshij hgxyfze
ftod vlryviqjb
lycs ovnseu
wloni hdo
yl queg
adexictvav uzgyn
u jafvuemw
yyo rkemhoorf
noxrsqdod sjsxfqxk
yrb brhxunazop
qqkieaexp zqtlnw
o wztkzsj
pfxlp zaphbse
nuru cgu
rleiet tygxa
d p
iwkwydxjob b
gmheq hrrnlmt
ebpyd rowznzgnh
jnqcphvwkk yhoqytvxqd
fsvog rd
svlax bnmkznpnge
vbzyb tccrnuk
90.81.0.236 8
dhivthkonl 5 jkndyaaufq nwknzbyb tbeckbc eieesshij qyvgb
r 1 pfxlp
hdpmbc 4 cj ptrcnstl pi wpuif
qtaceww 8 fwpjpqenie hvqmpo qyvgb mbmuccgjvt yl spfpz ftod tjwgjmil
gdzmvikjiq 5 kl gnsmes lycs xb ukbefigadr
tiapt 10 ydwjgfpdrg ebpyd rleiet dwuneonh imae svlax jfalxr fwpjpqenie kf cj
vzzmyobvoy 2 ftod uo
hvhnjz 6 aizdlaq nwknzbyb fdpth mcpqrgf uo odxpvatab
191.204.187.127 8
zfserdct 7 rleiet yyo dslqsjuw lycs acuzmuq dlsrnlh fwgokqg
pmkcisrjju 8 fdpth cvxmyajmf uolst ipuffds jfalxr fsvog hmnsptybct ypvtdveg
yyo 1 raplvroiuf
cegerojvc 1 ehovtn
hxszp 1 dwuneonh
dfvynrk 10 cke tbeckbc tjwgjmil lextffyzw ydwjgfpdrg uo zkcjfpnfn ypvtdveg gmheq svlax
xgske 1 dlsrnlh
ceys 10 ftod bd nuru oddz lycs eft xb wpuif uo eieesshij
113.90.222.216 7
ym 4 qsecd fwpjpqenie zssvjtbkf jfalxr
ezhlhiyh 4 cmnrne bd yswulbh fdpth
uzmtz 10 kf jfalxr xoxuuxy tbeckbc tjwgjmil x qqkieaexp fwgokqg yswulbh zssvjtbkf
rom 3 vbzyb naygip dlsrnlh
b 6 nuru qgj xoxuuxy qkzst ztygwg dlsrnlh
hvclobj 4 aizdlaq qqkieaexp jnqcphvwkk pi
hnqi 3 vjinpehodz mjnfoge noxrsqdod
84.79.103.208 10
odyyx 7 ccekxyuc ztygwg kl d wloni bd kf
fgytf 2 ydwjgfpdrg fwgokqg
kwuljjv 2 yswulbh mjnfoge
ev 3 cke dyox yrb
hdugzj 10 dlsrnlh zssvjtbkf mjnfoge yg lextffyzw aizdlaq lomxcojoow mcpqrgf durpdb jnqcphvwkk
zeklpe 9 lextffyzw lycs x adexictvav bd durpdb kl iwkwydxjob qyvgb
h 7 szd zkcjfpnfn oddz yrb vbzyb uayjyip svlax
lnd 4 fwgokqg yrb adexictvav lycs
nvf 2 ftod lextffyzw
wailhwap 6 cmnrne uayjyip adexictvav yyo kl ccekxyuc
255.189.64.165 5
hrjh 10 dyox noxrsqdod mt bd vbzyb yg rleiet wpuif wloni kf
ixkvdddajm 1 czdvpfcfb
akjv 9 dyox nuru dslqsjuw qyvgb hmnsptybct vjinpehodz rleiet lycs crnrlmad
kfqnynp 6 u ytppfqtmn yg mg fdpth cvxmyajmf
cwczjxwx 7 kf eft mcpqrgf cmnrne adexictvav aizdlaq fwgokqg
cwczjxwx 90.81.0.236 s
ev 84.79.103.208 wtz
hvhnjz 113.90.222.216 rowznzgnh
h 84.79.103.208 zonbdaa
h 84.79.103.208 bnmkznpnge
cegerojvc 255.189.64.165 c
