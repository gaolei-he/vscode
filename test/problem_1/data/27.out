No
Yes
No
No
Yes
No
